enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -285,15 +300,30 @@ class Meta:
 
     def can_redeem(self, learner_id, content_key):
         spent_amount = subsidy_client.amount_spent_for_learner(
-            subsidy_uuid = self.subsidy_uuid,
-            learner_id = learner_id,
-            )
+            subsidy_uuid=self.subsidy_uuid,
+            learner_id=learner_id,
+        )
         course_price = DiscoveryApiClient().get_course_price(content_key)
         if (spent_amount + course_price) < self.per_learner_spend_limit:
             return super().can_redeem(learner_id, content_key)
 
         return False
 
+    def credit_available(self, learner_id=None):
+        if self.remaining_credit(learner_id) > 0:
+            return True
+        return False
+
+    def remaining_credit(self, learner_id=None):
+        """
+        Returns the remaining redeemable credit for the policy.
+        """
+        spent_amount = subsidy_client.amount_spent_for_learner(
```

Comment:
[curious] Is the intent for `amount_spent_for_learner ` only to return the amount spent for transactions in a "committed" (i.e., fulfilled) state and without any reversals that would negate the transaction?

Reply:
That's how I'm hoping this endpoint in `enterprise-subsidy` should work.